### PR TITLE
v1.3.2 - Windows bootstrap cleanup

### PR DESCRIPTION
**Pull request by johndkraft**
Check for Admin rights or exit (#49)
Die with errors when they occur (ie: chef-client run, #33)
Use temporary folder under $ENV:TEMP (#56)
Install targeted version of omnitruck ChefDK, idempotently

### CODE REVIEW

**Comment by viebrock** [comment]
I might not make office hours today, but curious…  should I be looking to use this to update my windows laptop chefdk install?

**Comment by dougireton** [comment]
@johndkraft approved

File: bootstrap.ps1 [hunk 1]
```diff
@@ -49,9 +70,8 @@ $introduction = @"
 ### This bootstrap script will:
 
 1. Install the latest ChefDK package.
```
Comment (oifland) [inline bootstrap.ps1]: This is now a specifically targeted version

File: bootstrap.ps1 [hunk 2]
```diff
@@ -70,20 +90,49 @@ $berksfile | Out-File -FilePath $berksfilePath -Encoding ASCII
...
+}
 
 # Add ChefDK to the path
 $env:Path += ";C:\opscode\chefdk\bin"
 
-Set-Location $userChefDir
+Push-Location $tempInstallDir
 
 # Install the bootstrap cookbooks using Berkshelf
 berks vendor
```
Comment (oifland) [inline bootstrap.ps1]: If you stick this in, it'll make berks work when there's already a knife.rb and bomb out correctly when there's a problem:
```powershell
$env:BERKSHELF_CHEF_CONFIG = $chefConfigPath
berks vendor
if ( -not $? ) { Pop-Location;  die "Error running berks to download cookbooks." }
```

Comment (johndkraft) [inline bootstrap.ps1]: done

**Pull request merged**